data/hiera_options.yaml

```diff
@@ -0,0 +1,6 @@
+---
+lookup_options:
+  # Package lists should be replaced (not merged).
+  "^bacula::.*::packages":
+    merge:
+      strategy: first
```

Comment:
The `first` strategy in the default one so I don't think we need to configure it explicitly.

Reply:
In my own testing this was necessary, at least in the environment where I've done the testing. Otherwise the users' Hiera config may be applied to the module, breaking this functionality.

Reply:
Sorry, I can't reproduce the issue :sweat_smile:.  Can you show the output of `puppet lookup --explain` with and without this part so that I can see what is different on your setup?